.travis.yml

```diff
@@ -23,7 +23,12 @@ jobs:
       os: linux
       language: python
       env: TEST_SUITE=flake8
-    - stage: 'unit tests + documentation'
+    - stage: 'flake8 + documentation'
```

Comment:
These changes are unrelated, and they seem to be due to a merge conflict that was not resolved in the right way (they are undoing modifications that have been merged some time ago).